mbuild/lattice.py

```diff
@@ -435,8 +481,19 @@ def populate(self, compound_dict=None, x=1, y=1, z=1):
         Call Restrictions
         -----------------
         Called after constructor by user.
+
         """
+
         error_dict = {0: 'X', 1: 'Y', 2: 'Z'}
+
+        # Make sure neither x, y, z input is None
+        for replication_amount in x, y, z:
```

Comment:
Just curious @justinGilmer if this is preferred at all over `if None in [x, y, z]`?

Reply:
That seems like a much better way to handle it.
```suggestion
        if None in [x, y, z]:
            raise ValueError('Attempt to replicate None times. '
                             'None is not an acceptable replication '
                             'amount, 1 is the default.')
```